packages/services/policy/src/index.ts

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+import {
+  createServer,
+  registerShutdown,
+  registerTRPC,
+  reportReadiness,
+  startMetrics,
+} from '@hive/service-common';
+import * as Sentry from '@sentry/node';
+import { Context, schemaPolicyApiRouter } from './api';
+import { env } from './environment';
+
+async function main() {
+  if (env.sentry) {
+    Sentry.init({
+      serverName: 'schema',
+      enabled: !!env.sentry,
+      environment: env.environment,
+      dsn: env.sentry.dsn,
+      release: env.release,
+    });
+  }
+
+  const server = await createServer({
+    name: 'policy',
+    tracing: false,
+    log: {
+      level: env.log.level,
+      requests: env.log.requests,
+    },
+  });
+
+  registerShutdown({
+    logger: server.log,
+    async onShutdown() {
+      await server.close();
+    },
```

Comment:
```suggestion
    onShutdown: server.close,
```
as previous line `logger: server.log`

Reply:
I think it's not `bind`ed so we can't use it like that